setup.py

```diff
@@ -32,4 +32,4 @@
         "Operating System :: POSIX :: Linux",
         "Topic :: Software Development :: Compilers",
     ],
-)
```

Comment:
This was done in a browser and it just added the newline here, no way to prevent it :man_shrugging: